pkg/version/version.go

```diff
@@ -22,7 +27,22 @@ type Info struct {
 	Platform     string `json:"platform"`
 }
 
-// String returns info as a human-friendly version string.
-func (info Info) String() string {
+// String returns info as a human-friendly version string
+func (info *Info) String() string {
 	return info.GitVersion
 }
+
+// Get returns the version information of libgitops
+func Get() *Info {
```

Comment:
Why did you add this?
Actually, I'm not sure we should have this package at all under `pkg/` really, it kind of belongs under the `cmd/` example only. What do you think?

Reply:
This was copied over from Ignite's behavior to get the linter to pass (make use of all the vars set by ldflags). I agree it doesn't really make sense for a library, so I'll move it over to the example.